.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - *install_dependencies
-      - run:
+      - run: 
           name: Build
           command: |
               export REACT_APP_API_URL=https://summary-api.datamermaid.org;npm run build
```

Comment:
I'm also noticing that this doesn't match the name of the two environment variables you've generated.

Reply:
I run a local server to have better control on the api